packages/math/src/Matrix.ts

```diff
@@ -168,11 +168,11 @@ export class Matrix
      * Get a new position with the current transformation applied.
      * Can be used to go from a child's coordinate space to the world coordinate space. (e.g. rendering)
      *
-     * @param {PIXI.Point} pos - The origin
+     * @param {PIXI.IPointData} pos - The origin
      * @param {PIXI.Point} [newPos] - The point that the new position is assigned to (allowed to be same as input)
      * @return {PIXI.Point} The new point, transformed through this matrix
      */
-    apply(pos: IPoint, newPos?: Point): Point
+    apply(pos: IPointData, newPos?: Point): Point
```

Comment:
Could we change apply and applyInverse to `apply<P extends IPointData = Point>(pos: IPointData, newPos?:P):P`? That would allow us to have the same generic typing for `toLocal` and `toGlobal` on DisplayObject.

Reply:
`InteractionData.getLocalPosition()` could also benefit from this change (and have the `globalPos` parameter changed to an `IPointData`).